kitchen-vra.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'highline'
   spec.add_dependency 'rack', '~> 1.6' unless RUBY_VERSION.index('2.0.').nil?
   spec.add_dependency 'ffi-yajl', '~> 2.2.3' unless RUBY_VERSION.index('2.0.').nil?
-  spec.add_development_dependency 'bundler', '~> 1.7'
```

Comment:
In general we've just removed bundler as a dev dep since it's part of Ruby now

Reply:
oh good to know thanks!!